src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -484,8 +562,37 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def getMetaInfo(self, key: str) -> Optional[ConfigNodeMeta]:
+        if key in self._config_node_meta:
+            return self._config_node_meta[key]
+        return None
+
+    def getUseForwardInit(self, key) -> bool:
+        metaInfo = self.getMetaInfo(key)
+        return metaInfo.forward_init if metaInfo else False
+
     def have_forward_init(self) -> bool:
-        return _clib.ensemble_config.have_forward_init(self)
+        for config_key in self.alloc_keylist():
```

Comment:
Suggestion by linter makes sense here: `Use 'return any(self.getUseForwardInit(config_key) for config_key in self.alloc_keylist())'`